src/mixins/canvas_events.mixin.js

```diff
@@ -60,8 +85,7 @@
       functor(canvasElement, 'wheel', this._onMouseWheel);
       functor(canvasElement, 'contextmenu', this._onContextMenu);
       functor(canvasElement, 'dblclick', this._onDoubleClick);
-      functor(canvasElement, 'touchstart', this._onMouseDown, addEventOptions);
-      functor(canvasElement, 'touchmove', this._onMouseMove, addEventOptions);
+      functor(canvasElement, 'touchstart', this._onTouchStart, addEventOptions);
```

Comment:
do not register touchMove by default. is not used if not after a touchStart. Use specifct onTouchStart instead of _onMouseDown.